environment.yml

```diff
@@ -0,0 +1,245 @@
+  - partd=0.3.8=py36hf5c4cb8_0
+  - pcre=8.42=h378b8a2_0
+  - pexpect=4.6.0=py36_0
+  - pickleshare=0.7.5=py36_0
+  - pillow=5.3.0=py36hb68e598_0
+  - pip=10.0.1=py36_0
+  - pixman=0.34.0=hca0a616_3
+  - pluggy=0.7.1=py36h28b3542_0
+  - poppler=0.65.0=ha097c24_1
+  - poppler-data=0.4.9=0
+  - proj4=5.0.1=h1de35cc_0
+  - prometheus_client=0.4.0=py36_0
+  - prompt_toolkit=1.0.15=py36haeda067_0
+  - psutil=5.4.7=py36h1de35cc_0
+  - psycopg2=2.7.5=py36hdbc3d79_0
+  - ptyprocess=0.6.0=py36_0
+  - py=1.6.0=py36_0
+  - pycparser=2.19=py36_0
+  - pyepsg=0.3.2=py36_0
+  - pygments=2.2.0=py36h240cd3f_0
+  - pyopenssl=18.0.0=py36_0
+  - pyparsing=2.2.2=py36_0
+  - pyproj=1.9.5.1=py36h833a5d7_1
+  - pyqt=5.9.2=py36h655552a_2
+  - pysal=1.14.4.post1=py36_1
+  - pyshp=1.2.12=py36_0
+  - pysocks=1.6.8=py36_0
+  - pytest=3.8.2=py36_0
+  - python=3.6.6=hc167b69_0
+  - python-dateutil=2.7.3=py36_0
+  - python-snappy=0.5.3=py36h0a44026_0
+  - pytz=2018.5=py36_0
+  - pywavelets=1.0.1=py36h1d22016_0
+  - pyyaml=3.13=py36h1de35cc_0
+  - pyzmq=17.1.2=py36h1de35cc_0
+  - qt=5.9.6=h45cd832_2
+  - qtconsole=4.4.1=py36_0
+  - readline=7.0=h1de35cc_5
+  - requests=2.19.1=py36_0
+  - rtree=0.8.3=py36_0
+  - ruamel_yaml=0.15.46=py36h1de35cc_0
+  - scikit-image=0.14.0=py36h0a44026_1
+  - scikit-learn=0.20.0=py36h4f467ca_1
+  - scipy=1.1.0=py36h28f7352_1
+  - selenium=3.14.0=py36h1de35cc_0
+  - send2trash=1.5.0=py36_0
+  - setuptools=40.4.3=py36_0
+  - shapely=1.6.4=py36h20de77a_0
+  - simplegeneric=0.8.1=py36_2
+  - sip=4.19.8=py36h0a44026_0
+  - six=1.11.0=py36_1
+  - snappy=1.1.7=he62c110_3
+  - sortedcontainers=2.0.5=py36_0
+  - sqlalchemy=1.2.12=py36h1de35cc_0
+  - sqlite=3.25.2=ha441bb4_0
+  - streamz=0.3.0=py36_0
+  - tblib=1.3.2=py36hda67792_0
+  - terminado=0.8.1=py36_1
+  - testpath=0.4.2=py36_0
+  - thrift=0.11.0=py36h0a44026_0
+  - tk=8.6.8=ha441bb4_0
+  - toolz=0.9.0=py36_0
+  - tornado=4.5.3=py36_0
+  - tqdm=4.26.0=py36h28b3542_0
+  - traitlets=4.3.2=py36h65bd3ce_0
+  - urllib3=1.23=py36_0
+  - wcwidth=0.1.7=py36h8c6ec74_0
+  - webencodings=0.5.1=py36_1
+  - wheel=0.32.1=py36_0
+  - widgetsnbextension=3.4.2=py36_0
+  - xarray=0.10.9=py36_0
+  - xerces-c=3.2.1=h44e365a_0
+  - xz=5.2.4=h1de35cc_4
+  - yaml=0.1.7=hc338f04_2
+  - zeromq=4.2.5=h0a44026_1
+  - zict=0.1.3=py36_0
+  - zlib=1.2.11=hf3cbc9b_2
+  - intake=0.2.6=py_1
+  - intake-xarray=0.2.3=py_0
+  - hvplot=0.2.1=py_0
+  - parambokeh=0.2.3=py_0
+  - paramnb=2.0.4=py_0
+  - phantomjs=2.1.1=0
+  - pyct=0.4.5=py_0
+  - pyct-core=0.4.5=py_0
+  - pyviz_comms=0.6.0=py_0
+  - rise=5.3.0=py36_0
+  - bokeh=1.0.0dev6=pyhdd78535_0
+  - datashader=0.6.8=py_0
+  - geoviews=1.5.4a13=py_0
+  - geoviews-core=1.5.4a13=py_0
+  - holoviews=1.11.0a7=py_0
+  - panel=0.1.0a14=py_0
+  - param=1.8.0a10=py_0
+  - pyviz=0.9.13a0=py_0
+  - pip:
+    - affine==2.2.1
+    - msgpack==0.5.6
+    - rasterio==1.0.8
+    - rio-mucho==1.0.0
+    - rio-toa==0.3.0
+    - snuggs==1.4.2
+prefix: /Users/jstevens/miniconda3/envs/nasa_confirm
+
```

Comment:
Shouldn't be fully pinned like this; just pin pyviz and what we are actually relying on directly; the underlying packages can pin their own versions if need be.